csrc/preseg_passes/exact_mapped_extent_substitution.cpp

```diff
@@ -77,17 +84,23 @@ void exactMappedExtentSubstitution(Fusion* fusion) {
 
 void ExactMappedExtentSubstitutionPass::runPass(Fusion* fusion) {
   if (isDebugDumpEnabled(DebugDumpOption::PreSegmenterLogging)) {
-    debug() << "ExactLogicalDomainMap before " << name() << ":" << std::endl;
-    const auto mapped_sets = ExactLogicalDomainMap(fusion).getMappedSets();
-    debug() << mapped_sets.toString() << std::endl;
+    debug() << "DisjointSets before " << name() << ":" << std::endl;
+    IdModel id_model(fusion, false, false, false);
```

Comment:
```suggestion
    IdModel id_model(fusion, /*build_graphs=*/false);
```
The `allow_self_mapping` and `validate` arguments default to false anyway.